### DISCUSSION TITLE
Is 'Teams' the new term for 'Groups' ?

### DISCUSSION THREAD

**Discussion by adamdewey** [WeblateOrg/weblate#8398] [Q&A]
Hi, I've been reading through the documentation here and trying to get my head around Access Control: https://docs.weblate.org/en/weblate-4.3.2/admin/access.html

It keeps mentioning 'Groups' but in our version (4.11.2) I can't see any mention of Groups, I can only see something called 'Teams'. 

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/19318937/204746785-b77f1d4f-f9aa-4fb6-8e22-4c95bccd295b.png">

Was the name changed from 'Groups' to 'Teams' recently or are these different concepts?

Many thanks

**Comment by nijel** [comment]
The name was changed, but the behavior has changed as well. The "Groups" were hard-coded in Weblate and you could not (easily) define the own ones. The "Teams" can be created by the project admin.
